test/runtests.jl

```diff
@@ -12,6 +16,19 @@ begin
         include("interfaces/phase_space_layout.jl")
     end
 
+    @time @safetestset "coordinates" begin
+        include("coordinates.jl")
+    end
+
+    @time @safetestset "coordinate map" begin
+        include("coordinate_map.jl")
+    end
+
+    # TODO: move this to QEDbase
+    @time @safetestset "phase space layout" begin
+        include("interfaces/phase_space_layout.jl")
+    end
+
```

Comment:
This should not be here when merging, I imagine?

Reply:
These are the tests for the `patch_QEDbase.jl`. Since https://github.com/QEDjl-project/QEDbase.jl/pull/134 hasn't been merged yet and takes a bit more time due to CI problems, I would like to proceed with this PR. I suggest keeping the `TODO` comment here, to remind ourselves to remove this with the movement of `patch_QEDbase.jl`.